math/ibex-lib/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           waf 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+set real_version    2.8.5
+github.setup        ibex-team ibex-lib ibex-${real_version}
+version             ${real_version}
```

Comment:
Another commit reminded me of the correct way to handle this situation. All in one line:

`github.setup        ibex-team ibex-lib 2.8.5 ibex-
`

Reply:
Done.